### PR TITLE
Add membrane-curvature

### PR DESCRIPTION
**Pull request by IAlibay**

### CODE REVIEW

**Comment by IAlibay** [comment]
This is a good place to start a review @fiona-naughton - it's an ideal failure point a new user might come with.

**Comment by IAlibay** [comment]
Ok this one should be good for review @MDAnalysis/mdakits-reviewers

File: mdakits/membrane-curvature/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+#
+#------------------------------------------------------------
+# Required entries
+#------------------------------------------------------------
+## str: name of the project (the respository name)
+project_name: membrane-curvature
+
+## List(str): a link to the authors file (preferred) or a list of authors 
+authors:
+  - Estefania Barreto-Ojeda
```
Comment (fiona-naughton) [inline mdakits/membrane-curvature/metadata.yaml:22]: I'd suggest pointing to the AUTHORS file here instead:
```suggestion
  - https://github.com/MDAnalysis/membrane-curvature/blob/main/AUTHORS
```

**Comment by fiona-naughton** [comment]
Apologies for taking a while to get to this - but lgtm! Just one comment on the authors

**Review by fiona-naughton — APPROVED** [review]

**Comment by IAlibay** [comment]
Thanks @fiona-naughton !

**Pull request merged**